src/odemis/gui/cont/acquisition/fastem_acq.py

```diff
@@ -1197,6 +1205,91 @@ def __init__(
         # enable/disable calibration buttons if calibrating
         self._tab_data_model.is_calibrating.subscribe(self._on_is_calibrating)
 
+    def _on_visibility_btn(self, evt):
+        """Toggle the visibility of the calibration region based on the button state."""
+        current_sample = self._main_data_model.current_sample.value
+        focussed_view = self._main_tab_data.focussedView.value
+        # if current_sample and focussed_view:
```

Comment:
I think this if statement is needed and the below lines of code should be indented

Reply:
ah yes, that was left over from debugging. Should I update it to:
```suggestion
        if not current_sample and focussed_view:
            return
```